internal/component/otelcol/config_grpc.go

```diff
@@ -13,6 +13,8 @@ import (
 	otelextension "go.opentelemetry.io/collector/extension"
 )
 
+const DEFAULT_BALANCER_NAME = "pick_first"
```

Comment:
Idiomatically, it's unusual for constants in Go to be SNAKE_CASE like this. Can this be called `DefaultBalancerName` instead?

Reply:
Done :D